src/common/utils/arrays.py

```diff
@@ -0,0 +1,74 @@
+from typing import Any, Callable, Type
+
+from domain_classes.blueprint_attribute import BlueprintAttribute
+from domain_classes.dimension import Dimension
+
+
+def create_default_array(
+    dimension: Dimension,
+    blueprint_provider: Callable,
+    create_entity_class: Type[
+        Any
+    ],  # TODO use Type[CreateEntity]. can't use it now due to circular import. Needs refactoring.
+    blueprint_attribute: BlueprintAttribute,
```

Comment:
This parameter confused me at first, cause I didn't understand why you would need it. I see now that you only need blueprint_attribute.default. So perhaps instead do
```suggestion
    default: any,
```
That would be more intuitive, I think, but up to you